recipes-devtools/python/python3-powerrelay_git.bb

```diff
@@ -0,0 +1,36 @@
+DESCRIPTION = "REST API to control GPIO chips"
+HOMEPAGE = "https://github.com/prevas-dk/labgrid-powerrelay"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=20796caa814f193af92c180d146bb7ec"
+
+RDEPENDS_${PN} = " \
+    python3-click \
+    python3-aiohttp \
+    python3-trafaret \
+    python3-trafaret-config \
+    python3-aiohttp-jinja2 \
+    python3-yarl \
+    python3-multidict \
+"
+
+SRC_URI = " \
+    git://github.com/prevas-dk/labgrid-powerrelay.git;protocol=https;branch=master \
+    file://labgrid-powerrelay.service \
+    "
+SRCREV = "eced6843b34e59187278120401dcbd0f4b3b3016"
+
```

Comment:
Building this version based on the other PR's for the python dependencies I get the following run time error:
multidict<5.0,>=4.0 required. 
Installed is: 3.3.2
Just manually bumping to 4.5.2 on the target solves the issue. I would suggest to pin the multidict 4.5.2 in the powerrelay setups file.